### ISSUE TITLE
[Not a Bug]: Did u make pet finding ratio lower?

### ISSUE THREAD

**Issue by MicSwimming** [MelvorIdle/melvoridle.github.io#3113]
### Describe the bug

I know the ratio to get pets is low. But I did get pets easier before. I trained battle skill to 99 after DLC released, and I still got less pets. Even the dungeon pet like Yeti is somewhat harder to get. I have passed Frozen Cove over 700 times and got none, yet. I thought the ratio is 1/350? I got other dungeon pets normally before DLC. Just wonder if u changed the method about the criteria? Many thanks~

![20221224_MI_petissue_01](https://user-images.githubusercontent.com/121345245/209437990-d3edf971-fe31-4acb-a8be-20b37843a840.jpg)
![20221224_MI_petissue_02](https://user-images.githubusercontent.com/121345245/209437991-3f0173d4-945a-4c0a-ac23-b8903b2ee477.jpg)

### Screenshots

![20221224_MI_petissue_01](https://user-images.githubusercontent.com/121345245/209437998-a27fec6d-e6d8-47b3-be5c-014ab6067b01.jpg)
![20221224_MI_petissue_02](https://user-images.githubusercontent.com/121345245/209438001-a1fa062d-4bdb-47be-8155-ad08acd692e7.jpg)

### Which platforms are you experiencing this bug on?

- [ ] Chrome
- [ ] Firefox
- [ ] Edge
- [ ] Safari
- [ ] Mobile App (iOS)
- [ ] Mobile App (Android)
- [X] Steam
- [ ] Other (Please Specify)

### Which version of the game are you experiencing this bug on?

v1.1.1

### Game Subversion

?4999

### Are you using any mods?

No

**Comment by Draedon123** [comment]
Why are you posting a not bug into the bug system. Join the discord or subreddit to complain about poor rng there. Just saying, people have been known to reach levels above 120 before getting the pet (including me)

**Comment by MicSwimming** [comment]
Do u think I like to post this here? I apologize I post it here if bother u a lot. I just want to report a strange thing I met to stuff who made this game. And I click Error report button..... Then I have to sign up a website..... receive a code mail..... Why can't I just report my question and then get answer? And u suggest me join discord or subreddit!? WTF why I have to do that? Ok I agree the pets about battle skill are much harder to get. But how about Yeti? The ratio is 1/350. After I posted first report, I passed Frozen Cove 9xx before I slept w/o Yeti..... It's almost 3 times of 350! Thanks GOD I didn't buy the DLC! Maybe this game is fun before. But I think an idle game is just made gamers face rng!? lmfao! Let me know a game couldn't be more fun, let's just make gamers do it longer and harder to get something with doing the same boring thing!

![image](https://user-images.githubusercontent.com/121345245/209470399-6c4d1791-b19c-4e3b-b65c-08706722dd7c.png)
